package/src/components/Message/Message.tsx

```diff
@@ -288,7 +288,8 @@ const MessageWithContext = <
     },
   } = useTheme();
 
-  const actionsEnabled = message.type === 'regular' && message.status === 'received';
+  const actionsEnabled =
+    message.type === 'regular' && message.status === MessageStatusTypes.RECEIVED;
```

Comment:
Not for this PR I think, but adding to the Trello for later - We have `MessageLabel` in the JS client (it's not really clear to me why it's named Label - do you happen to remember? Should we use the same naming in the SDK?) for the `message.type` strings - I think it could be nice to do the same kind of refactor as you've done with the message status for those too. :)